Examples/analysis_default_regression.py

```diff
@@ -45,12 +39,19 @@ def main(args):
         type=str,
     )
     # add arguments: relative tolerance
+    # Identify whether the test is a restart test,
+    # if it is, use a 1e-12 tolerance for the restart checksum analysis
+    test_name = os.path.split(os.getcwd())[1]
+    if "_restart" in test_name:
+        default_tolerance = 1e-12
+    else:
+        default_tolerance = 1e-9
```

Comment:
This could be a one-liner:
```suggestion
    default_tolerance = 1e-12 if "_restart" in test_name else 1e-9
```